src/pudl_archiver/archivers/doeiraec.py

```diff
@@ -0,0 +1,54 @@
+"""Download DoE IRA Energy Community data.
```

Comment:
Little nit, but the o should be uppercase 

```suggestion
"""Download DOE IRA Energy Community data.
```

Reply:
oh haha ignore this RE: your comment above